test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua

```diff
@@ -137,6 +137,25 @@ local function ptuUpdateFuncNotValid(tbl)
     { "Base-4", "NewTestCaseGroup1", "NewTestCaseGroup2", "NewTestCaseGroup3" }
 end
 
+--[[ @ptuFuncEmptyArray: update table for PTU with empty array after removing unknown parameter
+--! @parameters:
+--! tbl - table for update
+--! @return: none
+--]]
+local function ptuFuncEmptyArray(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+        parameters = { unknownParameter }
+      }
```

Comment:
SubscribeVehicleData = {
        hmi_levels = { "BACKGROUND", "FULL", "LIMITED", "NONE" },
        parameters = { unknownParameter }
      }

Reply:
Seems HMILevel for application is "NONE"